index.html

```diff
@@ -0,0 +1,215 @@
+                                <li class="nav-link">Program</li>
+                                <li class="nav-link">join</li>
+                                <li class="nav-link">Sponsor</li>
+                                <li class="nav-link">News</li>
+                            </ul>
+                            <div class="campaign-donate mobile desktop">
+                                <p class="campaign-donate-text">Donate</p>
+                            </div>
+                            <div class="close">
+                                <img src="assets/icons/close-icon.png"
+                                    class="mobile"
+                                    id="menu-close" />
+                            </div>
+                        </div>
+
+                    </div>
+
+                </nav>
+            </header>
+            <section class="hero">
+                <div class="headline">
+                    <p>Your help goes a long way!</p>
+                    <h1 class="title-h1">Help us reach needy families and
+                        empower them get
+                        access to education. </h1>
+                </div>
+
+                <div class="card">
+                    <p class="headline-text">We aim to shape possibilities of
+                        under privileged
+                        children by providing a stepping stone to acquiring
+                        education and better living conditions.</p>
+                </div>
+
+                <q>The greatness of humanity is not in being human, but in being
+                    humane. – Mahatma Gandhi
+                </q>
+
+            </section>
+
+        </div>
+        <section class="values">
+            <h3>Our Values</h3>
+            <hr>
+            <div class="values-container">
+                <div class="bg">
+                    <ul class="values-reasons">
+                        <li><img
+                                src="/first-capstone-project/assets/images/integrity.png" /></li>
+                        <li><p>Integrity</p>
+                        </li>
+                        <li>
+                            <p>We focus on the entire customer lifecycle.</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="bg">
+                    <ul class="values-reasons" id="values">
+                        <li>
+                            <img
+                                src="/first-capstone-project/assets/images/responsive.png" />
+                            <li><p>Responsive</p></li>
+                            <li>
+                                <p>To the needs of the children in our target
+                                    group
+                                </p>
+                            </li>
+
+                        </li>
+
+                    </ul>
+
+                </div>
```

Comment:
Well done by implementing this part but I would suggest increading cards padding , increase card titles size and use the same color as in the design and finally reduce font size of the description 

![Capture d’écran du 2023-08-02 00-23-35](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/78356330-6b5e-4402-9485-02ec005b9aef)


![Capture d’écran du 2023-08-02 00-23-20](https://github.com/NoelLincoln/first-capstone-project/assets/59707859/2121bc1f-9019-417b-b06f-62faad9141d6)